### PR TITLE
Add Prisma ORM and Drizzle ORM

### PR DESCRIPTION
**Pull request by nikolasburk**
## Overview

This PR adds several new data services (using databases that are already part of the benchmark). Additionally, it adds the possibility to _execute these functions via the Node.js runtime_. This is enabled via a new (third) option in the **Location** section called **Test serverless function (Node)** which shows up only for those data services where a Node.js function is available.

- PlanetScale (Edge + Node)
  - Prisma ORM
  - Drizzle ORM
- Supabase (Node-only)
  - Prisma ORM
  - Drizzle ORM

For the new Supabase-based services, _only_ Node.js is available because Prisma and Drizzle connect to it via TCP which currently isn't possible in Vercel Edge Functions. 

As an example, here is the UI for the new service PlanetScale + Prisma ORM:

image

## TODOs

Two new environment variables need to be added for this PR to work properly:
- `PLANETSCALE_DATABASE_URL`: 
  - The full PlanetScale connection URL including the _database name_
  - It has the form: `mysql://DATABASE_USERNAME:DATABASE_PASSWORD@DATABASE_HOST/DATABASE_NAME?sslaccept=strict`
  - Example: `mysql://dcukom38k71ubg55ljqh:pscale_pw_password@aws.connect.psdb.cloud/vercel-benchmarks?sslaccept=strict`)
- `SUPABASE_DATABASE_URL`: 
  - The full Supabase connection URL (using a pooled connection)
  - It has the form: `postgres://DATABASE_USERNAME:DATABASE_PASSWORD@DATABASE_HOST:DATABASE_PORT/DATABASE_NAME`
  - Example: `postgres://postgres.oxlykxtvjloaxlizpwti:password@aws-0-us-east-1.pooler.supabase.com:5432/postgres`

## Adjust copy from "Vercel Edge Functions" to "Vercel Functions"?

I noticed that the Vercel docs have adopted the notion of [Vercel Functions](https://vercel.com/docs/functions) instead of Vercel Edge Functions recently. Also, with the introduction of Node.js-based functions in this repo, I was wondering whether the copy in the app should be adjusted from "Vercel Edge Functions" to "Vercel Functions" (and distinctions about the _runtime_ should be made on the page wherever relevant)? 

If that sounds useful, I'd be happy to take a stab at making copy suggestions in this (or another) PR so that the copy on the page feels consistent and sets expectations properly.

---

Let me know if there are any open questions or anything that you'd like to see changed in this PR :pray:

### CODE REVIEW

**Comment by nikolasburk** [comment]
Ah, one question: I noticed that the current Supabase route sends the query to a table called `employee_table` while in other, the table was called `employees`:

```ts
for (let i = 0; i < count; i++) {
  const response = await supabase
    .from("employee_table")
    .select("emp_no,first_name,last_name")
    .limit(10);
  data = response.data;
  console.log(data);
}
```

I implemented this with a table called `employees` for Supabase for Prisma and Drizzle. I guess this needs to be changed to `employees_table` so that the new routes work with the current Supabase DB?

**Comment by leerob** [comment]
This is looking awesome, thank you! Agree on the copy updates as well.

File: prisma-planetscale/prisma-client/index-browser.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+  makeStrictEnum,
+  Public,
+  getRuntime,
+} = require('./runtime/index-browser.js')
+
+
+const Prisma = {}
+
+exports.Prisma = Prisma
+exports.$Enums = {}
```
Comment (leerob) [inline prisma-planetscale/prisma-client/index-browser.js:16]: Should this be committed?

**Comment by nikolasburk** [comment]
> This is looking awesome, thank you! Agree on the copy updates as well.

Nice! I took a stab at those in the README and the app itself. Primarily, I ...
- changed occurrences of "Vercel Edge Functions" to just "Vercel Functions"
- clarified in a few spots that functions can run in Edge or Node runtimes to set better expectations for users

Let me know if that looks good or if you want to see different wording. 

image

image

The only thing that I can't change of course are the name of the repo and the deployment URL that still contain the word "Edge". Not sure if that's something you want to take a look at on your end?

**Review by leerob — APPROVED** [review]

**Pull request merged**